glide.lock

```diff
@@ -474,7 +474,7 @@ imports:
   - storage/v1alpha1
   - storage/v1beta1
 - name: k8s.io/apiextensions-apiserver
-  version: 63bd5e1c39483f0ef7fd275794d19a7cba19b764
+  version: 36be1826c980fa7ca82d91645e06e3157f159322
```

Comment:
@coreydaley We should pin this (and any other kube deps) to a 1.12 branch. 

Let's split this out into a separate PR and update this PR to only contain the changes around progress reporting in the sync controller.

Reply:
@adambkaplan Aren't they using 1.12 in the glide.yaml file? https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/glide.yaml ?